src/dotnet-core-uninstall/dotnet-core-uninstall.csproj

```diff
@@ -37,4 +37,23 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <Target Name="PublishMacOSUniversalBinary">
```

Comment:
who/what calls this target?

Reply:
probably here: https://github.com/dotnet/cli-lab/blob/25e91f1d39ad446131c0fe4bc998c5a33e766773/.vsts-ci.yml#L137C1-L144C33

You can tweak the .yml, then push your branch internal to trigger an official build against your branch, that's the only way to verify yaml changes